src/views/TechnicalAssistance/RequestForm/SubmittedRequest.tsx

```diff
@@ -0,0 +1,295 @@
+      <Divider />
+
+      {/* Basic request details */}
+      <h2 className="margin-top-3 margin-bottom-05">
+        {t('requestForm.steps.0.name')}
+      </h2>
+      {showEditSectionLinks && (
+        <div>
+          <UswdsReactLink to={`/trb/requests/${request.id}/basic`}>
+            {t('check.edit')}
+          </UswdsReactLink>
+        </div>
+      )}
+      <dl className="easi-dl margin-y-3">
+        <Grid row gap>
+          <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
+            <dt>{t('basic.labels.name')}</dt>
+            <dd>{request.name}</dd>
+          </Grid>
+          <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
+            <dt>{t('basic.labels.component')}</dt>
+            <dd>{request.form.component}</dd>
+          </Grid>
+          <Grid col={12}>
+            <dt>{t('basic.labels.needsAssistanceWith')}</dt>
+            <dd>{request.form.needsAssistanceWith}</dd>
+          </Grid>
+          <Grid col={12}>
+            <dt>{t('basic.labels.hasSolutionInMind')}</dt>
+            <dd>
+              {t(
+                `basic.options.${request.form.hasSolutionInMind ? 'yes' : 'no'}`
+              )}
+            </dd>
+            {request.form.hasSolutionInMind && (
+              <>
+                <dt>{t('basic.labels.proposedSolution')}</dt>
+                <dd>{request.form.proposedSolution}</dd>
+              </>
+            )}
+          </Grid>
+          <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
+            <dt>{t('basic.labels.whereInProcess')}</dt>
+            <dd>
+              {request.form.whereInProcess !== null &&
+                (request.form.whereInProcess === TRBWhereInProcessOption.OTHER
+                  ? `${t('basic.options.other')}: ${
+                      request.form.whereInProcessOther
+                    }`
+                  : t(
+                      `basic.options.whereInProcess.${camelCase(
+                        request.form.whereInProcess
+                      )}`
+                    ))}
+            </dd>
+          </Grid>
+          <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
+            <dt>{t('basic.labels.hasExpectedStartEndDates')}</dt>
+            <dd>
+              {request.form.hasExpectedStartEndDates &&
+              (request.form.expectedStartDate || request.form.expectedEndDate)
+                ? (() => {
+                    const start = request.form.expectedStartDate
+                      ? `${DateTime.fromISO(
+                          request.form.expectedStartDate
+                        ).toFormat('MM/dd/yyyy')} ${t('check.expectedStart')}`
+                      : '';
+                    const live = request.form.expectedEndDate
+                      ? `${DateTime.fromISO(
+                          request.form.expectedEndDate
+                        ).toFormat('MM/dd/yyyy')} ${t('check.expectedGoLive')}`
+                      : '';
+                    const and = start && live ? t('check.and') : '';
```

Comment:
I think these dates still need to be converted to UTC, per previous review comment

Reply:
Looks like this is resolved now that `formatDateUtc` is being used, resolving!